charms/knative-operator/src/manifests/auth_manifests.yaml.j2

```diff
@@ -22,7 +22,8 @@ aggregationRule:
 # automatically.
   - matchExpressions:
       - {key: serving.knative.dev/release, operator: Exists}
-rules: [] # Rules are automatically filled in by the controller manager.
+# Commenting out rules: [] due to https://github.com/gtsystem/lightkube/issues/32
+#rules: [] # Rules are automatically filled in by the controller manager.
```

Comment:
Did something change to make these a problem now, or were they broken before as well?

Reply:
Not sure what changed exactly, this didn't seem to be a problem in the past until now.

Reply:
I think this only matters when applying over an existing one, so maybe it just never came up before.  Either way, I think it works fine with this removed